test/bdd/pkg/walletjsonld/query.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package walletjsonld
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/hyperledger/aries-framework-go/pkg/wallet"
+)
+
+const (
+	sampleQueryByExFmt = `{
+                        "reason": "Please present your identity document.",
+                        "example": {
+                            "@context": [
+								"https://www.w3.org/2018/credentials/v1",
+								"https://www.w3.org/2018/credentials/examples/v1"
+                            ],
+                            "type": ["VerifiableCredential","UniversityDegreeCredential"],
+							"trustedIssuer": [
+								{
+									"required": true,
+									"issuer": "%s"
+								}
+							]
+                        }
+                	}`
+
+	presentationDefinitionFmt = `{
+					"id": "ec2f83c5-eac4-4d04-b41e-6636d6670a2e",
+					"input_descriptors": [
+						{
+							"id": "105b1d58-71f8-4d1e-be71-36c9c6f600c9",
+							"schema": [
+								{
+									"uri": "https://www.w3.org/2018/credentials#VerifiableCredential"
+								}
+							],
+							"constraints": {
+								"fields": [
+									{
+										"path": [
+											"$.issuer.id"
+										],
+										"filter": {
+											"type": "string",
+											"const": "%s"
+										}
+									}
+								]
+							}
+						}
+					]
+				}`
+)
```

Comment:
put it in `testdata` like other bdd tests

Reply:
done. Please see latest updates.